composer.json

```diff
@@ -28,29 +28,29 @@
     "source": "https://github.com/contao-community-alliance/translator"
   },
   "require": {
-    "php": ">=5.3",
-    "contao/core": ">=3.2,<3.6-dev",
-    "contao-community-alliance/composer-plugin": "~2.0",
-    "contao-community-alliance/event-dispatcher": "~1.0",
-    "contao-community-alliance/events-contao-bindings": "~3.0"
+    "php": ">=5.6",
+    "contao/core-bundle": "~4.4",
+    "contao-community-alliance/event-dispatcher": "~2.0@dev"
   },
   "require-dev": {
-    "phpcq/all-tasks": "~1.1"
+    "phpcq/all-tasks": "~1.1",
+    "contao/manager-bundle": "~4.4"
   },
   "autoload": {
-    "psr-0": {
-      "ContaoCommunityAlliance\\Translator": "src"
+    "psr-4": {
+      "ContaoCommunityAlliance\\Translator\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "ContaoCommunityAlliance\\Translator\\Test\\": "tests/"
     }
   },
   "extra": {
     "branch-alias": {
-      "dev-master": "2.0.x-dev"
+      "dev-feature/contao4": "2.1.x-dev"
```

Comment:
This is arguable.
The dependency on C4 makes it not installable on C3.5 and we said we wanted to increase the next minor to depend on the next major on Contao.
I am open to discuss this again however.